keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -188,21 +216,19 @@ metadata:
   finalizers:
   - cosi.io/finalizer [2]
 spec:
-  bucketName: [3]
-  generateBucketName: [4]
-  bucketClassName: [5]
-  secretName: [6]
-  protocol: [7]
-	accessMode: {"ro", "rw"} [8]
+  bucketPrefix: [3]
+  bucketClassName: [4]
+  secretName: [5]
+  protocol: [6]
+	accessMode: {"ro", "rw"} [7]
```

Comment:
This tab is failing the ./hack/verify.sh scripts.  should be converted to single spaces.